packages/genesys-spark-components/src/components/stable/gux-calendar/gux-calendar.service.ts

```diff
@@ -1,4 +1,6 @@
 import { capitalizeFirstLetter } from '@utils/string/capitalize-first-letter';
+import * as sparkIntl from '../../../genesys-spark-utils/intl';
+import { readRegionalDatesCookie } from '../../../i18n/check-regional-dates-cookie';
```

Comment:
Should we comment these line with a link to https://inindca.atlassian.net/browse/COMUI-2598 to make that issue more visible?